src/shared/store/cart/cart.effects.ts

```diff
@@ -69,10 +68,27 @@ export class CartEffects {
     )
   );
 
+  removeOrderedItemFromCart$ = createEffect((): Observable<Action> =>
+    this.actions$.pipe(
+      ofType(CartActions.clearCart),
+      switchMap(() => this.cartFacadeService.getCart().pipe(take(1))),
+      switchMap((cart: ICartItem[]) => {
+        const ids = cart.map((cart: ICartItem) => cart.id);
+
+        return this.cartService.clearCart(ids)
+          .pipe(
+            map(() => CartActions.cartCleared()),
+            catchError((error) => of(CartActions.errorCart({ error })))
```

Comment:
It's optional and just FYI: comma is not a blocker to merge the PR.
If you have some free minute when you fix other issues/comments, it'll not be hard to fix it here too :)
```suggestion
            catchError((error) => of(CartActions.errorCart({ error }))),
```